### ISSUE TITLE
CLI tools

### ISSUE THREAD

**Issue by Jefffrey** [datafusion-contrib/datafusion-orc#62]
Implement some CLI binaries for working with ORC files such as reading schema, getting stats, etc.

**Comment by klangner** [comment]
Is this issue taken?
I think it would be nice to have same examples how to use this library (or cli tools), since now it is not obvious how to use it. Maybe we could create list of tools/examples here (or as a separate issues?) so people could work on them.
From my use case I'm interested in tools for:
* Show metadata and schema
* Show top N records
* Show last N records

I would also be interested in reading them from S3, but that probably not in this project?

I'm also willing to help with those implementations

**Comment by waynexia** [comment]
A similar one is [`parquet-tools`](https://pypi.org/project/parquet-tools/), I used it several times when debugging with parquet files.

**Comment by klangner** [comment]
Yes, it looks nice. I can crate this cli tool (probably will start first with some simpler version first) if it fits this project and nobody is working on it yet.

**Comment by Jefffrey** [comment]
Hey @klangner I assigned it to myself initially because I did an initial commit as mentioned in the issue, but it isn't one of my priorities right now. Feel free to enhance the existing tool or add a separate one if you have a different use case :+1:
